pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -158,6 +158,8 @@ def qtProjectGenerator(package, argv, extra):
         help="File filter. A regex for matching additional files.")
     parser.add_argument('--exclude', default=[], action='append', dest="excludes",
             help="Package filter. A regex for excluding packages in QTCreator.")
+    parser.add_argument('--use-only', default=[], action='append', dest="useonly",
```

Comment:
I would add both arguments to a mutual exclusive group. Doesn't make any sense to me to use both at the same time. And i think it's undefined who wins this game..